kazoo/handlers/utils.py

```diff
@@ -10,18 +10,8 @@
 
 from collections import defaultdict
 
-import six
-
-if six.PY34:
-    import selectors
-else:
-    import selectors2 as selectors
-
-HAS_FNCTL = True
```

Comment:
`fcntl` is not a py27 thing (it was present then too). It is a runtime thing.

Reply:
Interesting. The docs says that it's not supported only in WASM. Are there people who use this lib with WASM?
I agree with keeping it, just curious

Reply:
Not that I know of....
It may simply be that much older versions of python did not have it.